test/integration-test.ts

```diff
@@ -1,5 +1,6 @@
 import { Wallet } from 'xpring-common-js'
 import { assert } from 'chai'
+import bigInt from 'big-integer'
```

Comment:
Is this a class? If so should it be capitalized?

Reply:
Sure the default BigInt is BigInt so we don't want to use that and the type is called BigInteger so I'm not sure we would want to use that either, but we can use something else.

Reply:
That makes sense enough for me. I'm fine with this as is.